lib/argument.js

```diff
@@ -47,6 +50,24 @@ var argumentValidation = {
     if (!callback || !_.isFunction(callback)) {
       throw new Error('acquireToken requires a function callback parameter.');
     }
+  }, 
+
+  validateUserCodeInfo : function(userCodeInfo) {
+     if (!userCodeInfo){
+        throw new Error('The userCodeInfo parameter is required');
+     }
+
+     if (!userCodeInfo.hasOwnProperty(userCodeResponseFields.DEVICE_CODE)){
```

Comment:
This is really minor, but I think `Object.prototype.hasOwnProperty.call(userCodeInfo, userCodeResponseFields.DEVICE_CODE)` is usually the preferred way to do this. At the moment you could get an error `Object userCodeResponseFields has no method 'hasOwnProperty'` if the object doesn't have Object on its prototype chain.